File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/For_Loops/factorial.cpp
// Printing a factorial

#include<iostream>
using namespace std;

int main()
{
    int n,f=1;
    cout<<"Enter the number whose factorial you want "<<endl;
    cin>>n;
    
    for(int i=n;i>=1;i--)
    {
        f=f*i;
    }
    cout<<"Factorial of "<< n <<" is : " << f;
}

File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/For_Loops/Factorial.java
// Printing a factorial

import java.util.*;

public class Factorial
{
    public static void main(String[] args)
     {
        
    int n,f=1;
    Scanner in=new Scanner(System.in);

    System.out.println("Enter the number whose factorial you want ");
    n=in.nextInt();
    
    for(int i=n;i>=1;i--)
    {
        f=f*i;
    }
    System.out.println("Factorial of "+ n +" is : "+ f);
}
}
